src/Iocoin/Transaction.cpp

```diff
@@ -0,0 +1,155 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Transaction.h"
+#include "../BinaryCoding.h"
+#include "../Hash.h"
+
+//#include <TrustWalletCore/TWBitcoin.h>
+
+#include <cassert>
+
+using namespace TW::Iocoin;
+std::vector<uint8_t> Transaction::getPrevoutHash() const {
```

Comment:
we prefer to use `Data` now
```suggestion
Data Transaction::getPrevoutHash() const {
```